features/bevragen/persoon/verblijfplaats/dev/fields-gba.feature

```diff
@@ -280,3 +280,94 @@ Rule: wanneer één of meerdere velden van een verblijfplaats buitenland wordt g
     | verblijfplaatsBinnenland.verblijfadres                                              |
     | verblijfplaatsBinnenland.datumVan,verblijfplaatsBinnenland.verblijfadres.huisnummer |
     | verblijfplaatsBinnenland.verblijfadres.regel1                                       |
+
```

Comment:
De titel van bovenstaand scenario klopt niet voor alle voorbeelden want in feite wordt alleen met het laatste voorbeeld echt een veld van verblijfplaats buitenland gevraagd. Bij de rest van de voorbeelden betreft het velden die ook of juist alleen betrekking hebben op een binnenlandsadres veld.